__tests__/08.gas-meter-api.ava.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * This test file is a copy of '01.basic-transactions.ava.ts' with the addition
+ * of a gas meter. The gas meter is used to track the amount of gas consumed.
+ *
+ * Note that the same tests will be run on both a local sandbox environment and
+ * on testnet by using the `test:sandbox` and `test:testnet` scripts in
+ * package.json.
+ */
+import {Worker, NEAR, NearAccount, GasMeter, Gas} from 'near-workspaces';
+import anyTest, {TestFn} from 'ava';
+
+const test = anyTest as TestFn<{
+  worker: Worker;
+  meter: GasMeter;
+  accounts: Record<string, NearAccount>;
+}>;
+
+test.beforeEach(async t => {
+  // Init the worker and start a Sandbox server
+  const meter = new GasMeter();
+  const worker = await Worker.init({tx_callbacks: [meter.tx_callback()]});
+
+  // Prepare sandbox for tests, create accounts, deploy contracts, etx.
+  const root = worker.rootAccount;
+  const contract = await root.devDeploy(
+    '__tests__/build/debug/status_message.wasm',
+    {initialBalance: NEAR.parse('3 N').toJSON()},
+  );
+  const ali = await root.createSubAccount('ali', {initialBalance: NEAR.parse('3 N').toJSON()});
+
+  // Assert some gas was burnt on account creation
+  t.notDeepEqual(meter.elapsed, Gas.from(0), `meter.elapsed: ${meter.elapsed.toString()}`);
```

Comment:
can we update this check to the format: 
```js
t.assert(meter.elapsed.gt(Gas.from(10_000_000_000_000)),  `meter.elapsed: ${meter.elapsed.toString()}`);
```
I think this checker contains more detail info about gas consumption